index.html

```diff
@@ -1,354 +1,369 @@
-      <a class="navbar-brand">
-        <img src="images/logo.png" height="150px" class="d-inline-block align-top" alt="" loading="lazy">
-      </a>
-      <!-- Adjust header -->
-      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-      </button>
-      <!-- Nav -->
-      <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <ul class="navbar-nav mr-auto">
-          <li class="nav-item active">
-            <a class="nav-link" href="#"><p style="font-size:2vw;">Home</p><span class="sr-only">(current)</span></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#About"><p style="font-size:2vw;">About</p></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#News"><p style="font-size:2vw;">News</p></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="team.html"><p style="font-size:2vw;">Team</p></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#Funding"><p style="font-size:2vw;">Funding</p></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="papers.html"><p style="font-size:2vw;">Papers</p></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="https://github.com/POSYDON-code/POSYDON/tree/main" target="_blank"><p style="font-size:2vw;">Code</p></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="./docs/index.html" target="_blank"><p style="font-size:2vw;">Docs</p></a>
-          </li>
-          <!-- <li class="nav-item">
-            <a class="nav-link" href="https://data.posydon.org" target="_blank"><p style="font-size:2vw;">Database</p></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="https://posydon.odahub.io" target="_blank"><p style="font-size:2vw;">Webapp</p></a>
-          </li> -->
-        </ul>
-      </div>
-    </nav>
-    
-    <div id="demo" class="carousel slide" data-ride="carousel">
-      <ul class="carousel-indicators">
-        <li data-target="#demo" data-slide-to="0" class="active"></li>
-        <li data-target="#demo" data-slide-to="1"></li>
-      </ul>
-      <div class="carousel-inner">
-        <div class="carousel-item active">
-          <img class="d-block mx-auto" src="images/carousel/webpage_photos.006.jpeg" alt="" style="width:100%; height: auto">
-          <div class="carousel-caption">
-            <p style="font-size:1.2vw;"><a href="https://ciera.northwestern.edu/gallery/binary-system-mass-transfer/" target="_blank" style="color: rgb(255,255,255)">Credit: NU Viz and CIERA - Matthew McCrory, Francesca Valsecchi, and Vicky Kalogera</a></p>
```

Comment:
Would it be possible to the old components in the carousel? Easiest might be to add some title with a link saying what the info is about (e.g. "Learn more: black hole accretion disks").

Reply:
Thanks for the suggestion. I've changed the format -- how's it look?